src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -2250,6 +2250,9 @@ class MenuItemKeyboardLayout final : public MenuItemSelection {
 // Colours submenu
 MenuItemSubmenu coloursSubmenu;
 
+MenuItemRuntimeFeatureSetting runtimeFeatureSettingMenuItem;
+MenuItemRuntimeFeatureSettings runtimeFeatureSettingsMenu;
```

Comment:
Yes and I'm not too happy about it. I copied the pattern from MIDIDevices menu so it might break too in the future. I hope once we fix initialization and allocation problems like this go away